Homework_8_classes_objects_fields_methods/MyAmazingRPG/src/Main.java

```diff
@@ -0,0 +1,54 @@
+public class Main {
+    public static void main(String[] args) {
+
+
+        Character sirius = new Character();
+        Character titanarm = new Character();
+
+
+        sirius.name = "SIRIUS";
+        sirius.health = 87;
+        sirius.strength = 19;
+        sirius.defense = 7;
+        sirius.criticalHit = sirius.strength + 20;
+        sirius.strengthUp = sirius.strength + 7;
+
+        titanarm.name = "TITANARM";
+        titanarm.health = 75;
+        titanarm.strength = 11;
+        titanarm.defense = 13;
+        titanarm.criticalHit = titanarm.strength + 15;
+        titanarm.strengthUp = sirius.strength + 6;
+
+        do {
+            titanarm.attack(sirius);
+            System.out.println("TITANARM attacked SIRIUS");
```

Comment:
This `sout` may be added to the `.attack()` method itself to keep the code cleaner (as you have access to the names of both participants)